tests/Predis/Command/Redis/CuckooFilter/CFMEXISTS_Test.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Redis\CuckooFilter;
+
+use Predis\Command\Redis\PredisCommandTestCase;
+
+class CFMEXISTS_Test extends PredisCommandTestCase
+{
+    /**
+     * {@inheritDoc}
+     */
+    protected function getExpectedCommand(): string
+    {
+        return CFMEXISTS::class;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function getExpectedId(): string
+    {
+        return 'CFMEXISTS';
+    }
+
+    /**
+     * @group disconnected
+     */
+    public function testFilterArguments(): void
+    {
+        $actualArguments = ['key', 'item1', 'item2'];
+        $expectedArguments = ['key', 'item1', 'item2'];
+
+        $command = $this->getCommand();
+        $command->setArguments($actualArguments);
+
+        $this->assertSameValues($expectedArguments, $command->getArguments());
+    }
+
+    /**
+     * @group disconnected
+     */
+    public function testParseResponse(): void
+    {
+        $this->assertSame(1, $this->getCommand()->parseResponse(1));
+    }
+
+    /**
+     * @group connected
+     * @return void
+     * @requiresRedisBfVersion >= 1.0
+     */
+    public function testExistsReturnsExistingItemsWithinCuckooFilter(): void
+    {
+        $redis = $this->getClient();
+
+        $redis->cfadd('key', 'item1');
+        $this->assertSame([1, 0], $redis->cfmexists('key', 'item1', 'item2'));
```

Comment:
Add more assertions, there are 3 possible cases:
1. All items exists
2. All items not exists
3. Partial items exists (one that you already added)